Merge Resolutions PR#40

jrodriguez0202 opened the pull request:
Did task on bug fixes +
-Fixed issue where null values caused exceptions, values that are needed are now replaced with "N/A"

jrodriguez0202 committed:
Merge Resolutions PR#40

laurenesco committed:
Removing junk files from git

laurenesco commented:
Searched "cat" in video games and clicked this entry, can you take a look into this

![image](https://user-images.githubusercontent.com/89123282/231013851-98b4f1aa-3e5b-49bf-89eb-245df92636e7.png)

laurenesco commented:
Also got this one; I searched "cat" on movies, then just switched to books and searched the exact same thing and got this:

![image](https://user-images.githubusercontent.com/89123282/231014214-446d54b2-2512-4c82-9fea-ff0193c21853.png)

laurenesco committed:
- Removed filters window on side of form; no longer going to be used.
- Resized form to final size.
- Set datagridview to be read only to avoid users editing and causing
  errors.

jrodriguez0202 committed:
fixed both errors

laurenesco commented:
Looks great, thank you!

laurenesco committed:
Delete .NETCoreApp,Version=v6.0.AssemblyAttributes.cs

laurenesco committed:
Delete MediaShelfApp.AssemblyInfo.cs

laurenesco committed:
Delete MediaShelfApp.GlobalUsings.g.cs

laurenesco committed:
Delete MediaShelfApp.AssemblyInfoInputs.cache

laurenesco committed:
Merge branch 'main' into jere_bugfix
